Documentation/Books/Manual/Views/ArangoSearch/Analyzers.md

```diff
@@ -1,4 +1,5 @@
-### Analyzers:
+ArangoSearch Analyzers
+======================
 
```

Comment:
On line 10:

>  A user then merely uses these analyzer names in ArangoSearch view configurations
 and AQL queries, e.g.

`e.g.` what? Is something missing here? Or can we just scratch it?

Reply:
Also removed.